src/passes/pass.cpp

```diff
@@ -369,6 +369,9 @@ void PassRegistry::registerPasses() {
   registerPass("print-stack-ir",
                "print out Stack IR (useful for internal debugging)",
                createPrintStackIRPass);
+  registerPass("propagate-globals-globally",
```

Comment:
Should this use `registerTestPass` to avoid it showing up in the `--help` output if it is only useful for tests? Or is it sometimes useful outside of tests as well?

Reply:
It is useful also for comparing stringref to imported strings.